homeassistant/components/kostal_plenticore/select.py

```diff
@@ -51,7 +51,7 @@ async def async_setup_entry(
     hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
 ) -> None:
     """Add kostal plenticore Select widget."""
-    plenticore: Plenticore = hass.data[DOMAIN][entry.entry_id]
+    plenticore = hass.data[DOMAIN][entry.entry_id]
```

Comment:
This change seems very unrelated to this PR?

Reply:
Was intended as a refactoring as other places in this integration do it the same way.
But I'm fine to keep it out of this PR